WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/blaze/CampaignStatus.kt

```diff
@@ -8,12 +8,12 @@ import org.wordpress.android.util.AppLog
 @Suppress("MagicNumber")
 
 enum class CampaignStatus(val status: String, @StringRes val stringResource: Int) {
+    InModeration("pending", R.string.campaign_status_in_moderation),
+    Scheduled("scheduled", R.string.campaign_status_scheduled),
     Active("active", R.string.campaign_status_active),
-    Completed("finished", R.string.campaign_status_completed),
     Rejected("rejected", R.string.campaign_status_rejected),
     Canceled("canceled", R.string.campaign_status_canceled),
-    Scheduled("scheduled", R.string.campaign_status_scheduled),
-    InModeration("created", R.string.campaign_status_in_moderation);
+    Completed("finished", R.string.campaign_status_completed);
```

Comment:
The new API exposes the following statues: 

![Screenshot 2024-03-04 at 17 37 02](https://github.com/wordpress-mobile/WordPress-Android/assets/2663464/2527dc60-3305-4c26-81d9-c16e774d396c)

The only change to what we were using before is that now `created` value has been replaced with `pending`. The rest of the returned values from the API are the same, I've just simply ordered them in "successive campaign life cycle" order.